schema/root.json

```diff
@@ -1,5 +1,6 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "$id": "https://lottie.github.io/lottie-spec/specs/schema/",
-    "$ref": "#/$defs/composition/animation"
+    "$id": "https://lottie.github.io/lottie-spec/specs/schema/000100",
+    "$ref": "#/$defs/composition/animation",
+    "$version": 100
```

Comment:
If we add $version at the top, I think we should add a check in schema-validator.py to enforce consistency